File: server/Models/Entities/Phase.cs
using System.Collections.Generic;

namespace server.Models.Entities
{
    public class Phase
    {
        public string PlayerId { get; set; }
        public List<CardCombination> CardCombinations { get; }
        public PhaseType PhaseType { get; set; }


        public Phase(string playerId, PhaseType phaseType, List<CardCombination> cardCombinations)
        {
            this.PlayerId = playerId;
            this.PhaseType = phaseType;
            this.CardCombinations = cardCombinations;
        }
    }
}

File: client-phase10/src/app/phase/phase.ts
import { CardCombination } from './cardCombination';

export interface Phase {
    playerId: string;
    phaseType: PhaseType;
    cardCombinations: CardCombination[];
}

export enum PhaseType {
    One = 1,
    Two,
    Three,
    Four,
    Five,
    Six,
    Seven,
    Eight,
    Nine,
    Ten
} 